test/edgeinfo.cc

```diff
@@ -0,0 +1,248 @@
+#include "baldr/edgeinfo.h"
```

Comment:
we already have so so many tests, the more we have the slower the build gets. we'd like to mitigate this over time by merging more together. is there another existing test where we can add these? imho we should take the edgeinfobuilder test, move it into this one and delete the edgeinfobuilder test. that way we avoid having 1 more binary to build and link

Reply:
>  imho we should take the edgeinfobuilder test, move it into this one and delete the edgeinfobuilder test

Will do :+1:

> that way we avoid having 1 more binary to build and link


Btw to my knowledge Google test allows to put all tests into single binary and then run them separately using CLI arguments. Have we ever considered it? I believe it should speed up builds a bit 🤔